core/dtn7/src/routing/erouting/ws_client.rs

```diff
@@ -0,0 +1,112 @@
+use super::Packet;
+use futures::channel::mpsc::{unbounded, UnboundedReceiver, UnboundedSender};
```

Comment:
why futures and not tokio?

Reply:
All channels were changed to tokio ones.